aioftp/client.py

```diff
@@ -562,6 +737,9 @@ class AsyncClientLister(AsyncListerMixin):
                 return cls
 
             async def __anext__(cls):
+                return await cls._anext()
+
+            async def _anext_MSLD_LIST(cls):
```

Comment:
Oh, I originally left that to implement NLST/CWD, but since you decided we don't need NLST/CWD, I will remove it.